config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml

```diff
@@ -62,7 +62,7 @@ presubmits:
       path_alias: k8s.io/kubernetes
     path_alias: sigs.k8s.io/kind
     decoration_config:
-      timeout: 40m
+      timeout: 50m
```

Comment:
We should bump k/k as well. This job is meant to give us the same experience as k/k presubmit.

Reply:
hahha you are 2 years ahead of the rest
https://github.com/kubernetes/test-infra/commit/ad9017ab1cb85d122e782781ab692e35610dd3cd

Reply:
let's have the same timeout in all of them